Release/7.0 minor fts updates (#3021)
couchbase/docs-server: modules/fts/pages/fts-query-string-syntax-field-scoping.adoc

* Prefixing with `+` places that item in the MUST portion of the boolean query.
* Prefixing with `-` places that item in the MUST NOT portion of the boolean query.

> Note, the above syntax is a simple and easy way to create xref:fts-supported-queries-compound-query.adoc[Compound Queries] in Search.

Reviewer: ```suggestion
TIP: You can use the preceding syntax to create xref:fts-supported-queries-compound-query.adoc[Compound Queries] in Search.
```
